src/main/java/com/limechain/runtime/RuntimeEndpoint.java

```diff
@@ -0,0 +1,20 @@
+package com.limechain.runtime;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * Holds all Runtime API endpoints' names.
+ */
+@Getter
+@AllArgsConstructor
+// TODO: Define all endpoints
+public enum RuntimeEndpoint {
+    CORE_EXECUTE_BLOCK("Core_execute_block"),
+    CORE_VERSION("Core_version"),
+    BLOCKBUILDER_CHECK_INHERENTS("BlockBuilder_check_inherents"),
+    METADATA_METADATA("Metadata_metadata"),
+    ;
```

Comment:
```suggestion
    METADATA_METADATA("Metadata_metadata");
```

Reply:
Hm I didn't know compiler doesn't complain about this. You can leave it if it's intentional.

Reply:
I tend to do this for larger WIP enums, where new line additions are expected toward the end. It is purely a style preference - for ease of adding new lines / moving lines around, to avoid the diff with the semicolon, etc.